catalog.html

```diff
@@ -184,7 +184,7 @@ <h2>каталог</h2>
 
         <li>
           <img
-          src="/img/A+.jpg"
+          src="/img/a+sayt.jpg"
```

Comment:
Не нужно использовать такие специфические символы.
Ну и написание не совсем то. Можно так например:
```suggestion
          src="/img/a-site.jpg"
```